tools/SourceKit/tools/sourcekitd-test/sourcekitd-test.cpp

```diff
@@ -935,17 +933,6 @@ static int handleTestInvocation(TestOptions Opts, TestOptions &InitOpts) {
     sourcekitd_object_t Args = sourcekitd_request_array_create(nullptr, 0);
     for (auto Arg : Opts.CompilerArgs)
       sourcekitd_request_array_set_string(Args, SOURCEKITD_ARRAY_APPEND, Arg);
-    if (ShouldIgnoreSourceInfo) {
-      // Ignore .swiftsourceinfo file when testing sourcekitd.
-      // .swiftsourceinfo for stdlib will be available when sourcekitd is tested,
-      // which may make some stdlib-depending sourcekitd tests volatile.
-      // We cannot append the flags when the compiler arguments are for clang
-      // invocation.
-      sourcekitd_request_array_set_string(Args, SOURCEKITD_ARRAY_APPEND,
-                                          "-Xfrontend");
-      sourcekitd_request_array_set_string(Args, SOURCEKITD_ARRAY_APPEND,
-                                          "-ignore-module-source-info");
-    }
```

Comment:
This is awesome!
Because of this, `sourcekitd-test` was not compatible to older versions of SourceKit (because they don't understand `-ignore-module-source-info` frontend option).